tutorial/csharp.async/Tutorial/Work.cs

```diff
@@ -0,0 +1,249 @@
+/**
```

Comment:
Generated thrift files should not be part of the PR.

Reply:
delete files.

Reply:
This looks like it's still here.